.editorconfig

```diff
@@ -0,0 +1,33 @@
+# https://editorconfig.org
+# Editor Config for WebMemex
+# Download an extension for editor config to allow your IDE
+# to follow the rules of the project
+
+# Used to indicate the file is in root of the project
+root = true
+
+# Styling applicable to all forms of files
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+# Styling applicable to the file extension given
+[*.js, *.jsx]
+# This is due to the fact that github allows only 119 columns of space to view
+max_line_length = 119
```

Comment:
I understand github diff works on this length, I'm not discussing the number, I'm dicussing having any number there.
If tomorrow (real life story) I need to put an svg encoded icon somewhere I do want it on one line and I don't care if you have to scroll.
I'm not really worried of a dev writing a very very long line, why would it do that? So I consider this rule more a possible issue than a safeguard.